cypress/support/commands.js

```diff
@@ -360,3 +360,11 @@ Cypress.Commands.add('seedCircleMember', (name, userId, type = 1, level) => {
 			}
 		})
 })
+
+/**
+ * Fail the test on the initial run to check if retries work
+ */
+Cypress.Commands.add('testRetry', () => {
```

Comment:
Is this actually used anywhere? Or do you want to add it for using it later on?

Reply:
The latter. you can put `cy.testRetry()` at the end of a test and it will fail on the first two runs.

This way you can check if retries can actually pass or will always fail. I thought this might be useful to keep around. but i can als always add it back if i need it.

Reply:
Ah, great. I'm fine with keeping it :blush:

Reply:
Nice one